src/functions/utils/slackBot.ts

```diff
@@ -20,17 +23,37 @@ export class SlackBot {
 
   private registerCommands(): void {
     const slackBotCommand = process.env.SLACK_BOT_COMMAND as string;
+    const currentStage = process.env.STAGE as string;
+
     this.app.command(slackBotCommand, async ({ command, ack, respond }: SlackCommandMiddlewareArgs) => {
       await ack();
-      await respond(`Hello, <@${command.user_id}>! 👋`);
-
-      console.log("Slack command executed:", {
-        userId: command.user_id,
-        command: slackBotCommand,
-        details: command,
-        timestamp: new Date().toISOString(),
-      });
-      console.log(`Your stage is ${process.env.STAGE}`);
+
+      let userName: string;
```

Comment:
This is only used in the try block below and not changed. Define as const inside the block. Linter should have complained about this as well. Why did it not?